RELEASE_NOTES.md

```diff
@@ -1,5 +1,118 @@
+- Improved `port` resource performance: adding more specific search while using `ss` command.
+- Updated the `inspec init plugin` command with the following changes:
+  - The values of flags passed to the `inspec init plugin` command are now wrapped in double quotes instead of single quotes.
+  - Template files are now ERB files.
+  - The `activator` flag replaces the `hook` flag, which is now an alias.
+
+#### Bug Fixes
+
+- Fixed an error when using profile dependencies and require_controls.
+- Fixed the `windows_firewall_rule` resource when it failed to validate more than one rule.
+- The `http` resource response body is now coerced into UTF-8.
+- Modified the `windows_feature` resource to indicate if a feature is enabled rather than just available.
+- `file` resource `more_permissive_than` matcher returns nil instead of throwing an exception when the file does not exist.
+- `inspec detect --no-color` now returns color-free output.
+
+### Slow Resource Report
+
+Chef Infra Client now includes a `--slow-report` flag that shows the 10 slowest running resources in a Chef Infra Client run to help you troubleshoot and optimize your cookbooks. This new flag also takes an argument for the number of resources to list if you'd like to see additional resources included in the output. Our next release of Chef Workstation will include the ability to set this flag in Test Kitchen to allow testing for slow resources in the development process.
+
+#### Example Output
+
+```text
+Starting Chef Infra Client, version 17.2.12
+Patents: https://www.chef.io/patents
+resolving cookbooks for run list: ["test"]
+Synchronizing Cookbooks:
+  - test (0.0.1)
+Installing Cookbook Gems:
+Compiling Cookbooks...
+Converging 1 resources
+Recipe: test::default
+  * file[/tmp/foo.xzy] action create (up to date)
+
+Running handlers:
+
+Top 1 slowest resource:
+
+resource           elapsed_time cookbook recipe  source
+------------------ ------------ -------- ------- ----------------------------------------
+file[/tmp/foo.xzy] 0.015114     test     default test/recipes/default.rb:2:in `from_file'
+
+  - Chef::Handler::SlowReport
+Running handlers complete
+Chef Infra Client finished, 0/1 resources updated in 03 seconds
+```
+
+Add a slow resources report to chef-client - https://github.com/chef/chef/pull/11642
+
+### Improved YAML Recipe Support
+
+Chef Infra Client now supports YAML recipes that end in `.yaml` in addition to those ending in `.yml`. If a `.yml` and `.yaml` recipe of the same name is present Chef Infra Client will now fail as there is no way to determine which recipe should be loaded in this case.
+
+#### Improved Reporting to Automate
+
+Chef Infra Client run reporting to Automate now respects attribute `allowlist` and `denylist` configurations set in the `client.rb` configuration. This change allows users to limit the data sent to their Automate servers to prevent indexing sensitive data, or to reduce the necessary storage space on the Automate server.
```

Comment:
```suggestion
Chef Infra Client run reporting to Automate now respects attribute `allowlist` and `denylist` configurations set in the `client.rb`. This change allows users to limit the data sent to their Automate servers to prevent indexing sensitive data or to reduce the necessary storage space on the Automate server.
```